test/models/backend_api_test.rb

```diff
@@ -30,6 +30,16 @@ def test_default_api_backend
     assert @backend_api.valid?
   end
 
+  test '.orphans should return backend apis that do not belongs to any service' do
+    FactoryBot.create(:service)
+    service_to_delete = FactoryBot.create(:service)
+    orphan_backend_api = service_to_delete.backend_api_configs.first.backend_api
+
+    service_to_delete.destroy!
+
+    assert_equal [orphan_backend_api], BackendApi.orphans
```

Comment:
Well there is only one backend api in the DB so this test does not really compare anything